gateway/client-switchover/README.md

```diff
@@ -0,0 +1,422 @@
+kafka-consumer-groups --describe --all-groups \
+  --bootstrap-server kafka-destination:9193 \
+  --command-config destination-cluster.config > /tmp/destination-offsets.txt
+  
+# Validate the difference.
+diff /tmp/source-offsets.txt /tmp/destination-offsets.txt
+```
+
+#### 3. Verify all topics are mirrored
+```bash
+# Get source kafka topics.
+kafka-topics --list --bootstrap-server kafka-source:9093  --command-config source-cluster.config | sort > /tmp/source-topics.txt
+
+# Get mirrored topics in the destination.
+kafka-mirrors \
+  --bootstrap-server kafka-destination:9193 \
+  --command-config destination-cluster.config \
+  --list \
+  --link source-to-destination-link | sort > /tmp/destination-topics.txt
+
+# Validate that any difference is expected (test or internal topics only).
+diff /tmp/source-topics.txt /tmp/destination-topics.txt
+```
+
+### Step 2: Traffic cutover to Green cluster
+
+#### 1. Patch loadbalancer to point to the Green deployment
+- Patch the loadbalancer to change the label selectors as well as the `targetPort` for the Kafka bootstrap listener.
+- Modify the `targetPort` in the below patch command to point to the port of the Green gateway route endpoint.
+```
+kubectl patch service confluent-gateway-switchover-lb -n confluent --type='json' -p='[
+  {"op": "replace", "path": "/spec/selector/app", "value": "confluent-gateway-green"},
+  {"op": "replace", "path": "/spec/ports/0/targetPort", "value": 9696}
+]'
+```
+
+####  2. Test message consumption from new Gateway setup [Loadbalancer pointing to Green deployment]
+- You should be able to consume the test messages that were produced in `test-topic` as part of the Cluster Linking verification step.
+```
+kafka-console-consumer \
+  --bootstrap-server gateway.example.com:9595 \
+  --consumer.config destination-cluster.config \
+  --topic test-topic \
+  --from-beginning
+```
+
+### Step 3: Promote Mirror Topics
+#### NOTE: Please modify the `bootstrap-server` config appropriately for all commands in this section.
+
+- Promote mirrors to make them writable. Modify below command to include required topic names
+```bash
+kafka-mirrors --promote \
+  --topics test-topic \
+  --bootstrap-server kafka-destination:9193 \
+  --command-config destination-cluster.config
+```
+
+- Verify promotion by producing messages:
+```
+kafka-console-producer \
+  --bootstrap-server gateway.example.com:9595 \
+  --producer.config destination-cluster.config \
+  --topic test-topic
+```
+
+### Step 4: Monitor and Validate
+- Monitor for 15-30 minutes. Watch for:
+  - Producer reinitialization (ProducerFencedException should spike then stabilize).
+  - Consumer duplicate processing (should spike then decrease).
+  - Application error rates.
+
+### Step 5: Scale down Blue deployment
+
+- Scale down Blue deployment to 0 replicas. Goal is to retain for 24-48 hours in case of rollback.
+```
+kubectl scale deployment confluent-gateway-blue -n confluent --replicas=0
+```
+
+## Client Impact Summary
```

Comment:
If we have validated, we can keep this section. else let it remain part of internal docs only.

Reply:
I had taken it from the wiki page. Have removed it.